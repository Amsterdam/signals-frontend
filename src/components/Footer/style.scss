.footer-component {
  padding: 20px;
}
