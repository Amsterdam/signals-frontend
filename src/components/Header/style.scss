@import '~stijl/dist/scss/ams-colorpalette';
@import '~stijl/dist/scss/ams-breakpoints';

.header-component {
  .mainlogo {
    text-decoration: none;

    .logoset .screen-logo {
      width: 20px;
    }
  }

  .links {
    margin-top: 1.125rem;

    // TODO: remove this li {} block and uncomment the one below it once npm package stijl v2.9.2 available

    li {
      text-align: right;
    }
      /*
      li {
        text-align: left;

        @include media-breakpoint-up(sm) {
          text-align: right;
        }
      }
      */
  }
}
