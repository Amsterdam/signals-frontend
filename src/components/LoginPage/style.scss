.login-page {
  margin-top: 150px;
}
