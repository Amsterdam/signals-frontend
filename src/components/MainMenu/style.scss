@import '~amsterdam-stijl/dist/scss/ams-colorpalette';

.main-menu-component {
  background-color: $ams-top-menu;
}
