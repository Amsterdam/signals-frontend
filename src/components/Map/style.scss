.map-component {
  #mapdiv {
    position: relative;
    width: 100%;
    height: 450px;
  }

  &__loading {
    background-color: rgba(0, 0, 0, .2);
    display: block;
    width: calc(100% - 30px); // padding left and right of grid
    height: 100%;
    position: absolute;
    z-index: 1001;
  }
}
