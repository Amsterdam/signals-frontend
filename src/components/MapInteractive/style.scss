.map-component {
  margin-top: 5px;

  #mapdiv {
    position: relative;
    width: 100%;
    height: 450px;
  }

  .embed-search {
    // pull the search input outside of the map
    // top: -50px;
    // left: 0;
    max-width: 100%;
    // width: 100%;

    input[type="text"] {
      height: inherit;
    }

    button {
      display: none;
    }

    &__auto-suggest {
      width: 100%;
    }

    .invoer {
      box-shadow: none;
    }
  }

  &__loading {
    background-color: rgba(0, 0, 0, .2);
    display: block;
    width: calc(100% - 30px); // padding left and right of grid
    height: 100%;
    position: absolute;
    z-index: 1001;
  }
}
