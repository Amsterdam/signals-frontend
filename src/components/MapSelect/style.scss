@import "config.scss";

.map-component {
  .map-container {
    position: relative;
    width: 100%;
    height: 450px;
  }

  &__loading {
    background-color: $ams-lichtgrijs;
    display: block;
    width: calc(100% - 30px); // padding left and right of grid
    height: 100%;
    position: absolute;
    z-index: 1001;
  }

  &.write {
    .object-marker:hover {
      filter: drop-shadow(0 0 2px $ams-zwart);
    }
  }

  .zoom-control, .legend-control, .loading-control, error-control {
    border: 2px solid rgba($ams-zwart, .02);
    background-clip: padding-box;
  }

  .zoom-control {
    background-color: #FEC813;
    color: $ams-zwart;
    font-family: Avenir Next LT W01, arial, sans-serif;
    font-size: 16px;
    line-height: 22px;
    padding: 13px;
  }

  .legend-control {
    background-color: $ams-wit;
    color: $ams-zwart;
    font-family: Avenir Next LT W01, arial, sans-serif;
    font-size: 16px;
    line-height: 16px;
    min-width: 240px;
  }

  .legend-header {
    padding: 14px;
    color: $ams-zwart;

    background-color: #F5F5F5;;

    background-image: url("../../shared/images/Chevron-Down.svg");
    background-position: right;
    background-repeat: no-repeat;
    background-size: 16px 16px;
    background-origin: content-box;

    &:hover, &:focus {
      background-color: $ams-pinkish-grey;
      color: $ams-zwart;
      fill: $ams-zwart;
      outline: none;
    }
  }

  .legend-content {
    background-color: $ams-wit;
    padding: 5px 14px;
  }

  .legend-content-item {
    display: flex;
    align-items:center;
    height: 28px;
    line-height: 28px;
    margin: 10px 0;
  }

  .legend-content-text {
    margin-left: 8px;
  }

  .legend-control {
    &.is-closed {
      .legend-header {
        background-image: url("../../shared/images/Chevron-Top.svg");
      }

      .legend-content {
        display: none;
      }
    }
  }


  .loading-control {
    background-color: $ams-wit;
    font-family: Avenir Next LT W01, arial, sans-serif;
    font-size: 16px;
    line-height: 22px;
    padding: 13px 17px;
  }


  .error-control {
    background-color: $ams-wit;
    font-family: Avenir Next LT W01, arial, sans-serif;
    font-size: 16px;
    line-height: 22px;
    padding: 13px 17px;
  }


  .hide {
    display: none;
  }
}
