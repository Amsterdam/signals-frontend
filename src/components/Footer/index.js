import React from 'react';

import './style.scss';

function Footer() {
  return (
    <div className="footer-wrapper no-print">
      <div className="row bg-darkgrey footer">
        <footer className="container footer-component">
          <div className="row">
            <div className="col-12">
              <h2>
                Lukt het niet om een melding te doen?
              </h2>
              <p>
                Bel het Gemeentelijk informatienummer: 14 020 <br />
                op werkdagen van 08.00 tot 18.00 uur.
              </p>
            </div>
          </div>
        </footer>
      </div>
      <div className="container grid-below-footer">
        <div className="row">
          <div className="col-12">
            <nav>
              <ul className="links horizontal left">
                <li>
                  <a href="https://www.amsterdam.nl/privacy/">
                    <span className="linklabel">
                      Privacy
                    </span>
                  </a>
                </li>
              </ul>
            </nav>
          </div>
        </div>
      </div>
    </div>
  );
}

export default Footer;
