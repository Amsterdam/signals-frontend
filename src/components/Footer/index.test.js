import React from 'react';
import { shallow } from 'enzyme';

import Footer from './index';
import './style.scss';

describe('<Footer />', () => {
  it('should render correctly', () => {
    const wrapper = shallow(
      <Footer />
    );
    expect(wrapper).toMatchSnapshot();
  });
});
