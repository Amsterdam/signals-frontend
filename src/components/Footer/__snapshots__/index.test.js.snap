// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> should render correctly 1`] = `
<div
  className="footer-wrapper"
>
  <div
    className="row bg-darkgrey footer"
  >
    <footer
      className="container footer-component"
    >
      <div
        className="row"
      >
        <div
          className="col-12"
        >
          <h2>
            Lukt het niet om een melding te doen?
          </h2>
          <p>
            Bel het Gemeentelijk informatienummer: 14 020 
            <br />
            op werkdagen van 08.00 tot 18.00 uur.
          </p>
        </div>
      </div>
    </footer>
  </div>
  <div
    className="container grid-below-footer"
  >
    <div
      className="row"
    >
      <div
        className="col-12"
      >
        <nav>
          <ul
            className="links horizontal left"
          >
            <li>
              <a
                href="#Amsterdam"
              >
                <span
                  className="linklabel"
                >
                  Privacy
                </span>
              </a>
            </li>
            <li>
              <a
                href="#Amsterdam"
              >
                <span
                  className="linklabel"
                >
                  Cookies
                </span>
              </a>
            </li>
            <li>
              <Connect(LocaleToggle) />
            </li>
          </ul>
        </nav>
      </div>
    </div>
  </div>
</div>
`;
